client/src/components/tabs.vue

```diff
@@ -58,25 +58,27 @@ export default class Tabs extends Vue {
 
 </script>
 
-<style lang="sass" scoped>
-#tabs-landing-comp
-  max-width: 1264px !important
-
-.tab-items
-  border-bottom-left-radius: 6px !important
-  border-bottom-right-radius: 6px !important
-
-.active-tab
-   background-color: #003366
-
-.upper-left-border
-   border-top-left-radius: 6px
-   min-height: 58px
-   max-height: 58px
-
-.upper-right-border
-   border-top-right-radius: 6px
-   min-height: 58px
-   max-height: 58px
-
+<style lang="scss" scoped>
+@import '@/assets/scss/theme.scss';
+#tabs-landing-comp {
+  max-width: 1140px !important;
+  padding: 0 !important;
+}
+.tab-items {
+  border-bottom-left-radius: 6px !important;
```

Comment:
https://vuetifyjs.com/en/styles/border-radius/

Reply:
Will update!

Reply:
Not sure they have a specific size for 6px BR as it is relational, so i used the library and just applied the radius size.

Reply:
I used Vuetify class "rounded" (ie, medium) in a couple of places, which is 4px.